src/features/home/HomeProjectPending.tsx

```
@@ -0,0 +1,27 @@
+import ContentsCard from './components/ContentsCard';
+
+const HomeProjectPending = () => {
+  const {data: projectList} = useQuery(userQueries.groupsPending());
+
+  return (
+    <ContentsCard title="수락 대기중인 프로젝트" arrowLink="/projects">
+      <div className="flex flex-col gap-2 mt-[14px] text-left">
+        {projectList && projectList.length > 0 ? (
+          projectList.map(project => (
+            <div className="p-3.5 bg-[#EEECFF] rounded-md">
+              <span className="text-sm text-[#5F4AFF] font-medium">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)